specs/disciplina/witness/api/witness.yaml

```diff
@@ -216,6 +216,84 @@ paths:
           schema:
             $ref: '#/definitions/TxValidationError'
 
+  /transactions/money:
```

Comment:
It should be either a filter parameter to `/transactions` endpoint (like `type=(money|pub|all)`, with default value `all`), if we don't want to separate publications and transactions completely, or it should be two endpoints `/transactions` (existing one) and `/publications`, otherwise. In the latter case, a single type `TransactionInfo` should be split into two types representing info about money transaction and info about publication.

Your current approach is unnecessarily verbose.

Reply:
Fair, adding a filter to `/transactions` should work.